src/harness/reference_models/pre_iap/pre_iap.py

```diff
@@ -0,0 +1,55 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
Please name the directory and file `pre_iap_filtering` to match with the name commonly used.

Reply:
Changed   to performPreIapFiltering